content/src/Blip.Api.Template/Startup.cs

```diff
@@ -52,20 +62,44 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<ILogger>(new LoggerConfiguration()
                      .ReadFrom.Configuration(Configuration)
                      .Enrich.WithMachineName()
-                     .Enrich.WithProperty(APPLICATION_KEY, Constants.PROJECT_NAME)
+                     .Enrich.WithProperty(APPLICATION_KEY, Models.Constants.PROJECT_NAME)
                      .Enrich.WithExceptionDetails()
                      .CreateLogger());
 
             // BLiP services registration
-            services.DefaultRegister(settings.BlipBotSettings.Authorization);
+            var builder = new BlipClientBuilder();
+            builder.UsingAccessKey(settings.BlipBotSettings.Identifier, settings.BlipBotSettings.AccessKey)
+                .UsingRoutingRule(Lime.Messaging.Resources.RoutingRule.Instance)
+                .WithChannelCount(2);
+
+            var blipClient = builder.Build();
+            services.AddSingleton<ISender>(blipClient);
```

Comment:
`services.DefaultRegister` already does this and more, I'd rather enable it to receive a TCP ISender (or let the user choose between both protocols) instead